src/main/java/frc/robot/commands/LedSwitchCommand.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.LedSubsystem;
+
+public class LedSwitchCommand extends CommandBase {
```

Comment:
What is this command supposed to do? If it is only supposed to toggle a variable, it should be an InstantCommand (examples in RobotContainer) that calls a method within LedSubsystem